packages/file-upload/src/FileUpload.tsx

```diff
@@ -313,13 +313,7 @@ export const fileInputStyles = ({
 					backgroundColor: boxPalette.systemSuccessMuted,
 					borderColor: boxPalette.systemSuccess,
 			  }
-			: disabled
-			? {
-					cursor: 'not-allowed',
-					opacity: 0.3,
-					background: 'none',
-			  }
 			: undefined),
 
-		'&:focus': packs.outline,
+		...(disabled && packs.disabled),
```

Comment:
I have updated how disabled are applied here so we can still show disabled styles when the file upload component is invalid/valid like our other form components. 

I've also removed the background colour reset when the component disabled, as that seems inconsistent with our other form components?

Screen shot of the update

<img width="1263" alt="Screen Shot 2022-10-19 at 9 36 10 am" src="https://user-images.githubusercontent.com/6265154/196564705-74e521b7-b58d-418c-9f70-c62401111176.png">